ocaml/libs/open-uri/open_uri.ml

```diff
@@ -53,6 +53,25 @@ let with_open_uri uri f =
              (Uri.to_string uri)
           )
   )
+  | Some "https" -> (
+      let verify_cert =
+        Option.value ~default:(Stunnel_client.pool ()) verify_cert
```

Comment:
I like that users are forced to be explicit about the verification they need if they require something else other than pool verification